.github/workflows/test.yml

```diff
@@ -98,8 +94,10 @@ jobs:
           sed -i -e 's/^PREFIX=.*$/PREFIX=\/usr/g' Makefile
           make
           sudo make install
-      - name: Install dependencies
-        run: |
-          WITH_VTERM=1 bundle install
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+          bundler-cache: true
```

Comment:
This automatically runs bundled install and caches the result, so we don't need a separate step for it. Putting it after libvterm installation to make sure we have all the dependencies needed for gems.